src/dask_awkward/lib/core.py

```diff
@@ -1400,7 +1400,10 @@ def _getitem_tuple(self, where):
             return self._getitem_outer_int(where)
 
         elif isinstance(where[0], str):
-            return self._getitem_outer_str_or_list(where)
+            first, rest = where[0], where[1:]
```

Comment:
@jpivarski at some point talked about the possibility of reordering these elements to shuffle "fields" before "indexes", which would be more general than this, which only account for "more than one". I think in this context, "fields" is always str and "indexes" can be slice, int, ints or bools. Of course, this is also complicated by ellipsis, but I think we already accounted for that before calling here.